x-pack/legacy/plugins/siem/public/components/ml_popover/hooks/use_job_summary_data.tsx

```diff
@@ -11,40 +11,38 @@ import { KibanaConfigContext } from '../../../lib/adapters/framework/kibana_fram
 import { hasMlUserPermissions } from '../../ml/permissions/has_ml_user_permissions';
 import { MlCapabilitiesContext } from '../../ml/permissions/ml_capabilities_provider';
 
-type Return = [boolean, Job[] | null];
+type Return = [boolean, Job[]];
 
 export const getSiemJobsFromJobsSummary = (data: Job[]) =>
   data.reduce((jobs: Job[], job: Job) => {
     return job.groups.includes('siem') ? [...jobs, job] : jobs;
   }, []);
 
-export const useJobSummaryData = (jobIds: string[], refetchSummaryData = false): Return => {
-  const [jobSummaryData, setJobSummaryData] = useState<Job[] | null>(null);
+export const useJobSummaryData = (jobIds: string[] = [], refetchSummaryData = false): Return => {
+  const [jobSummaryData, setJobSummaryData] = useState<Job[]>([]);
   const [loading, setLoading] = useState(true);
   const config = useContext(KibanaConfigContext);
   const capabilities = useContext(MlCapabilitiesContext);
   const userPermissions = hasMlUserPermissions(capabilities);
 
   const fetchFunc = async () => {
-    if (jobIds.length > 0) {
-      if (userPermissions) {
-        const data: Job[] = await jobsSummary(jobIds, {
-          'kbn-version': config.kbnVersion,
-        });
+    if (userPermissions) {
+      const data: Job[] = await jobsSummary(jobIds, {
+        'kbn-version': config.kbnVersion,
+      });
 
-        // TODO: API returns all jobs even though we specified jobIds -- jobsSummary call seems to match request in ML App?
-        const siemJobs = getSiemJobsFromJobsSummary(data);
+      // TODO: API returns all jobs even though we specified jobIds -- jobsSummary call seems to match request in ML App?
```

Comment:
Appreciate the TODO. We will look at this at a later time, but even if later they fix this, your code shouldn't break which is a great thing.